src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -30,11 +31,11 @@ LoadingPriority = 1
 
 TranscodeVideo = MPEGTS-H264-AC3
 TranscodeAudio = WAV
-#MaxVideoWidth = 3840
-#MaxVideoHeight = 2160
+MaxVideoWidth = 3840
+MaxVideoHeight = 2160
 SupportedVideoBitDepths = 8,10
-H264Level41Limited = true
-#H265Level41Limited = true
+H264Level41Limited = false
```

Comment:
This is a work around.
This renderers support until 5.1 H264 and h265 at HD, and 4.1 at UHD.

But the problem is that we don't implement, and never did, a correct DLNA implementation, and the Samsung renderers, like @squadjot one, are not able to transcode with UMS.
So that's the less worst situation, IMO.
I advertised the users and they was fine with it, as they could finally get their movies play, it was better than not, also if some will fail.
That's said, H265 with a 5.1 profile or superior are not popular at all.